tests/fuzz/fuzz-close_tx.c

```diff
@@ -0,0 +1,90 @@
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+#include <tests/fuzz/libfuzz.h>
+
+#include <bitcoin/pubkey.h>
+#include <bitcoin/chainparams.h>
+#include <bitcoin/script.h>
+#include <bitcoin/tx.h>
+#include <ccan/tal/tal.h>
+#include <common/amount.h>
+#include <common/close_tx.h>
+#include <common/setup.h>
+#include <common/utils.h>
+#include <wire/wire.h>
+
+void init(int *argc, char ***argv)
+{
+	common_setup("fuzzer");
+	chainparams = chainparams_for_network("bitcoin");
+}
+
+void run(const uint8_t *data, size_t size)
+{
+	const uint8_t *wire_ptr;
+	size_t wire_max, min_size, script_size;
+	struct bitcoin_txid txid;
+	uint32_t vout;
+	struct amount_sat funding, to_us, to_them, dust_limit, max;
+	const uint8_t *our_script, *their_script, *funding_script;
+	struct pubkey *pk1, *pk2;
+
+	/* create_close_tx wants:
+	 * - 3 scripts: 3 * N bytes
+	 * - 1 txid: 32 bytes
+	 * - 1 u32: 4 bytes
+	 * - 4 amount_sat: 4 * 8 bytes
+	 *
+	 * Since the output scripts size is not very restricted (and the shutdown_scriptpukeys
+	 * are not by the spec either), we also try to vary their length.
```

Comment:
what does 'not by the spec' mean?

Reply:
Am i wrong that the spec does not restrict the size of the `shutdown_scriptpubkeys` ?

Reply:
afaik that is correct. I'd suggest rewording this to 

```
Since output script (and shutdown_scriptpubkey) size is not restricted...
```